frontend/packages/dev-console/src/components/edit-application/__tests__/EditApplicationForm.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { formikFormProps } from '@console/shared/src/test-utils/formik-props-utils';
+import PipelineSection from '@console/pipelines-plugin/src/components/import/pipeline/PipelineSection';
+import GitSection from '../../import/git/GitSection';
+import BuilderSection from '../../import/builder/BuilderSection';
+import DockerSection from '../../import/git/DockerSection';
+import ImageSearchSection from '../../import/image-search/ImageSearchSection';
+import IconSection from '../../import/section/IconSection';
+import AppSection from '../../import/app/AppSection';
+import AdvancedSection from '../../import/advanced/AdvancedSection';
+import { CreateApplicationFlow } from '../edit-application-utils';
+import EditApplicationForm from '../EditApplicationForm';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+describe('EditApplicationForm', () => {
+  const componentProps = {
+    ...formikFormProps,
+    appResources: {},
+  };
+
+  it('should hide git & pipeline sections for container edit form', () => {
```

Comment:
Can you break this into two separate tests? This is for checking if the git and pipeline sections are hidden in container edit form so I think checking if these sections are present in git and docker forms should be done in a separate test. Similarly you can do this for other tests. Because I think what you are testing should match the test case as well as that way it becomes easier to debug if something breaks.

Reply:
Updated.